packages/itwin/map-layers/src/ui/widget/MapLayerManager.tsx

```diff
@@ -12,6 +12,7 @@ import { BentleyError, compareStrings } from "@itwin/core-bentley";
 import { BackgroundMapProvider, BackgroundMapType, BaseMapLayerSettings, ImageMapLayerSettings } from "@itwin/core-common";
 import { ImageryMapTileTree, IModelApp, MapLayerSources, NotifyMessageDetails, OutputMessagePriority } from "@itwin/core-frontend";
 import { ToggleSwitch } from "@itwin/itwinui-react";
+import * as MapLayersFormats from "@itwin/map-layers-formats";
```

Comment:
why are we importing the entire module like this?

Reply:
removed